src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/PerformerNameAndAddress.java

```diff
@@ -0,0 +1,104 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+/**
+ * Example: NAD+SLA+A2442389:902++DR J SMITH'<br>
+ * Example: NAD+SLA+++ST JAMES?'S UNIVERSITY HOSPITAL'
+ */
+@Getter
+@Setter
+@Builder
+@RequiredArgsConstructor
+public class PerformerNameAndAddress extends Segment {
+
+    private static final String KEY = "NAD";
+    private static final String QUALIFIER = "SLA";
+    public static final String KEY_QUALIFIER = KEY + "+" + QUALIFIER;
+    private static final int PERFORMING_NAME_INDEX_IN_EDIFACT_STRING = 4;
+    private static final int PERFORMER_ID_INDEX_IN_EDIFACT_STRING = 2;
+    private static final int PERFORMER_CODE_INDEX_IN_EDIFACT_STRING = 3;
+
+    @NonNull
```

Comment:
I think this decorator could be removed. Doesn't make sense when `identifier` can be null (or empty) if the segment/section is related to a performing organisation

Reply:
Done in fe1470c